bfex/components/scrapper/scrapper.py

```diff
@@ -0,0 +1,102 @@
+from bs4 import BeautifulSoup
+import requests
+import json
+import re
+
+base_url = "https://www.ualberta.ca/science/about-us/contact-us/faculty-directory/"
+
+ORCID = 0
+RESEARCHID = 1
+PROFILE = 2
+
+class Scrapp(object):
+    def __init__(self):
+        self.title = ""
+        self.text = ""
+        self.date = ""
+        self.meta_data = {}
+
+    def set_title(self, title):
+        self.title = title
+
+    def set_text(self, text):
+        self.text = text
+
+    def set_date(self, date):
+        self.date = date
+
+    def add_meta(self, hash, meta):
```

Comment:
Hash is confusing, as you aren't currently hashing things into it. Youre just passing a string. Maybe call it attr ?